spec/models/search/tag_query_spec.rb

```diff
@@ -156,4 +158,14 @@
     results = tag_query.search_results
     expect(results.size).to eq 5
   end
+
+  before do
```

Comment:
FYI, this `before` isn't scoped, so it'll be called before all of the tests in this file, not just the single test that needs it.

Reply:
I can clean this up while on [AO3-5581](https://otwarchive.atlassian.net/browse/AO3-5581).

Reply:
Fixed in #3512.